### ISSUE TITLE
Add project reference (using absolute paths) creates incorrect relative path

### ISSUE THREAD

**Issue by thomaschrstnsn** [fsprojects/Forge#122]
## Expected Behaviour

A project structure as such:
```
/Users/thchristensen/src/fsdemo/src/
  -  Library/Library.fsproj 
   - ConsoleApp/ConsoleApp.fsproj
```

Referencing `Library` from `ConsoleApp` (using VSCode Ionide-fsharp 2.2.10) issues the following command:

```
forge add project -n /Users/thchristensen/src/fsdemo/src/Library/Library.fsproj -p /Users/thchristensen/src/fsdemo/src/ConsoleApp/ConsoleApp.fsproj --no-prompt
```
## Current Behavior
Part of the change to the `ConsoleApp.fsproj` is:
```
    <ProjectReference Include="Library/Library.fsproj">
```
Expected path to be:
```
    <ProjectReference Include="../Library/Library.fsproj">
```

## Steps to Reproduce (for bugs)
1. Create two new projects, one console app and one classlib in a subfolder to the project (`/src`)
2. Try to reference the class lib from the console app

## Your Environment

* Version used: Forge shipped with vscode-ionide-fsharp 2.2.10
* Environment name and version (F#, .Net): Mono OSX (homebrew)

**Comment by ErikSchierboom** [comment]
I'm working on this problem!

**Comment by Krzysztof-Cieslak** [comment]
Fix released in `1.0.0-alpha011`
